fastavro/_write.pyx

```diff
@@ -5,24 +5,19 @@
 # Apache 2.0 license (http://www.apache.org/licenses/LICENSE-2.0)
 
 import json
-
 import datetime
 import decimal
 import uuid
-import time
 from binascii import crc32
-from collections import Iterable, Mapping
 from libc.time cimport tm, mktime
 from cpython.int cimport PyInt_AS_LONG
 from cpython.tuple cimport PyTuple_GET_ITEM
-from libc.string cimport memset
-from os import urandom, SEEK_SET
+from os import urandom
 from zlib import compress
-import numbers
 
 from fastavro import const
-from .const import DAYS_SHIFT
-from ._six import utob, long, is_str, iterkeys, itervalues, iteritems, mk_bits
+from .validate import validate
```

Comment:
It shouldn't technically matter, but we should change this to `from ._validate import validate` to match the other imports.